src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx

```diff
@@ -15,14 +14,31 @@ import { DisplayField } from './DisplayField';
 export const formatDomains = (domains?: Domain[]): string =>
   (domains ?? []).map((domain: Domain) => domain.domain).join(', ');
 
+const GetAnLEI = (
+  <Link href='https://www.gleif.org/en/about-lei/get-an-lei-find-lei-issuing-organizations'>
+    GLEIF
+  </Link>
+);
+
 const defaultDescription = (
   <>
     To update the email domains for your financial institution,{' '}
     <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update financial institution profile: Update email domain'>
       email our support staff
     </Link>
     . To update any other data in this section, contact your Local Operating
-    Unit (LOU) or visit <Links.GLIEF /> to identify your LOU.
+    Unit (LOU) or visit {GetAnLEI} to identify your LOU.
+  </>
+);
+
+export const updateInstitutionDescription = (
+  <>
+    To update the email domains for your financial institution,{' '}
+    <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update financial institution profile: Update email domain'>
+      email our support staff
+    </Link>
+    . To update any other data in this section, contact your Local Operating
+    Unit (LOU) or visit {GetAnLEI} for a list of LOUs.
```

Comment:
I think this language hasn't been updated everywhere in figma yet, but I think this is supposed to be "visit GLIEF to identify your LOU"? Take a look at the figma under View your financial institution profile.

![Screenshot 2024-06-11 at 10 14 49 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/3e09d9b3-0d3d-4887-8d5f-30c8033ec777)


![Screenshot 2024-06-11 at 10 15 21 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/904347fe-2396-4b69-bdda-7a7a167681ef)